modules/appeals_api/config/schemas/10182.json

```diff
@@ -46,18 +46,50 @@
       "additionalProperties": false,
       "properties": {
         "homeless":            { "type": "boolean" },
-        "representativesName": { "type": "string", "maxLength": 120 },
-        "address": { "$ref": "#/definitions/nodCreateAddress" }
+        "address":             { "$ref": "#/definitions/nodCreateAddress", "minLength": 5, "maxLength": 165 },
+        "phone":               { "$ref": "#/definitions/nodCreatePhone" },
+        "emailAddressText":    { "$ref": "#/definitions/nodCreateEmail" },
+        "representativesName": { "type": "string", "maxLength": 120 }
       },
-      "required": [ "homeless", "address" ]
+      "required": [ "homeless", "phone", "emailAddressText" ]
     },
 
 
     "nodCreateAddress": {
       "type": "object",
-      "properties": { "zipCode5": { "type": "string", "pattern": "^[0-9]{5}$", "minLength": 5, "maxLength": 5 } },
       "additionalProperties": false,
-      "required": [ "zipCode5" ]
+      "properties": {
+        "addressLine1":            { "type": "string" },
+        "addressLine2":            { "type": "string" },
+        "addressLine3":            { "type": "string" },
+        "city":                    { "type": "string" },
+        "stateCode":               { "$ref": "#/definitions/nodCreateStateCode" },
+        "countryName":             { "type": "string" },
+        "zipCode5":                { "type": "string", "pattern": "^[0-9]{5}$", "minLength": 5, "maxLength": 5 },
+        "internationalPostalCode": { "type": "string" }
+      },
+      "required": [ "addressLine1", "city", "countryName", "zipCode5" ]
+    },
+
+
+    "nodCreatePhone": {
+      "$comment": "the phone fields must not exceed 25 chars, when concatenated",
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "countryCode":     { "type": "string", "pattern": "^[0-9]+$", "minLength": 1, "maxLength": 1 },
```

Comment:
This maxLength is incorrect according to the pattern. I'm not sure how much of an edge case it would be, but if someone's main contact info was still in Germany (because they were stationed there recently?) then they wouldn't be able to enter `49` as the country code given the `maxLength: 1`.

Reply:
Yeah, that was a derp on my part. So:

- Isle of Man country code: `44-1624`

Safe to say 6 digits max?

Reply:
`Thanks to the international phone numbering plan (ITU-T E. 164), phone numbers cannot contain more than 15 digits. The shortest international phone numbers in use contain seven digits.`

Reply:
Just to be clear, the Isle of Man's country code is `44` (for the country of United Kingdom) and its area code is `1624` (for the area of Isle of Man). The note about ITU-T E. 164 is useful though :) I believe we limit the combined phone number (including extension) to 20 characters for HLR - we might be able to get away with the same here, too.